io.github.chkr1011.mqttMultimeter.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: io.github.chkr1011.mqttMultimeter
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+
+command: mqttMultimeter
+
+modules:
+  - name: mqttMultimeter
+    buildsystem: simple
+    build-commands:
+      - unzip mqttMultimeter-linux-x64.zip
+      - chmod +x mqttMultimeter
+      - install -D mqttMultimeter /app/bin/mqttMultimeter
+      - install -D io.github.chkr1011.mqttMultimeter.png /app/share/icons/hicolor/256x256/apps/io.github.chkr1011.mqttMultimeter.png
+      - install -D io.github.chkr1011.mqttMultimeter.desktop /app/share/applications/io.github.chkr1011.mqttMultimeter.desktop
+      - install -D io.github.chkr1011.mqttMultimeter.metainfo.xml /app/share/metainfo/io.github.chkr1011.mqttMultimeter.metainfo.xml
+
+    sources:
+    - type: file
+      url: https://github.com/chkr1011/mqttMultimeter/releases/download/v1.9.0.290/mqttMultimeter-linux-x64.zip
```

Comment:
how will this work on aarch64? Why not building from source instead?

Reply:
I will remove the aarch64 support. Right now, I cannot build from source because it needs a windows runner to compile the application because it uses a special feature (resources) which requires that (at the moment).